### ISSUE TITLE
Missing #include <mc_rtc/gui.h> for Stabilizer.cpp and ModelPredictiveControl.cpp

### ISSUE THREAD

**Issue by chrisyeew** [jrl-umi3218/lipm_walking_controller#7]
During compilation it was giving me errors that elements related to GUI were not declared in the scope. Fixed it by `#include <mc_rtc/gui.h>`, but I don't know where you would want to put it globally so that it works.

**Comment by arntanguy** [comment]
Use `topic/mc_rtc_master`.
I'll merge to master ASAP ;)

**Comment by arntanguy** [comment]
I've just opened PR https://github.com/jrl-umi3218/lipm_walking_controller/pull/8, which is @stephane-caron's latest master but with catkin_free compilation.

**Comment by chrisyeew** [comment]
Hm, switching to `topic/mc_rtc_master` gives me
```
MCUDPControl: /usr/include/eigen3/Eigen/src/Core/DenseBase.h:261: void Eigen::DenseBase<Derived>::resize(Eigen::Index, Eigen::Index) [with Derived = Eigen::Block<Eigen::Matrix<double, -1, -1>, -1, -1, false>; Eigen::Index = long int]: Assertion `rows == this->rows() && cols == this->cols() && "DenseBase::resize() does not actually allow to resize."' failed.
```
Going back to `master` with my hack inclusion of `#include <mc_rtc/gui.h>` fixes it.

**Comment by arntanguy** [comment]
Fixed by #8
